test/WebJobs.Script.Tests/UtilityTests.cs

```diff
@@ -38,6 +41,70 @@ public class TestPocoEx : TestPoco
 
     public class UtilityTests
     {
+        [Fact]
```

Comment:
Would probably be good to have a test to ensure min and max are properly applied (although that's pretty straight forward)